core/src/main/java/com/nisovin/magicspells/util/MenuData.java

```diff
@@ -0,0 +1,7 @@
+package com.nisovin.magicspells.util;
+
+import org.bukkit.entity.LivingEntity;
+
+public record MenuData(SpellData spellData, int page) {
+
+}
```

Comment:
Unnecessary to have as a separate class. Should be an inner class in `PlayerMenuSpell`.